app/layout.tsx

```diff
@@ -24,17 +21,9 @@ export default function RootLayout({
   children: React.ReactNode;
 }>) {
   return (
-    <html lang="en" data-theme="dark" style={{ colorScheme: 'dark' }}>
-      <body
-        className={`${manrope.className} bg-surface-surface-bg text-base antialiased`}
-      >
-        <Suspense>
-          <Providers>
-            <Navbar />
-            <NextThemesProvider>{children}</NextThemesProvider>
-            <Footer />
-          </Providers>
-        </Suspense>
+    <html lang="en">
+      <body className={`${manrope.className} text-base antialiased`}>
```

Comment:
```suggestion
      <body className={`${manrope.className} text-base text-text-high-em antialiased`}>
```

We should add this default text color.